app/views/pages/contact.html.erb

```diff
@@ -27,7 +27,7 @@
   </div>
   <div class="body">
     <p>
-      The Practical Dev would love to hear from you!
+      <%= community_qualified_name %> would love to hear from you!
     </p>
     <p>
       Email: <a href="mailto:<%= SiteConfig.default_site_email %>"><%= SiteConfig.default_site_email %></a> 😁
```

Comment:
@lightalloy not sure if you've pulled the latest master here, but this should have been SiteConfig.email_addresses[:default] on master. (maybe there will or won't be conflicts, not sure 🤷🏽‍♀️)